test/hotspot/jtreg/serviceability/sa/TestJhsdbJstackWithVirtualThread.java

```diff
@@ -37,7 +37,7 @@
  * @test
  * @bug 8369505
  * @requires vm.hasSA
- * @requires (os.arch == "amd64" | os.arch == "x86_64" | os.arch == "aarch64" | os.arch == "riscv64")
+ * @requires (os.arch == "amd64" | os.arch == "x86_64" | os.arch == "aarch64" | os.arch == "riscv64" | os.arch == "ppc64" | os.arch == "ppc64le")
```

Comment:
Is there still an arch that is unsupported? Otherwise the whole line can be dropped, no?

Reply:
I think it can be removed.